src/boundary/Multigrid.h

```diff
@@ -8,31 +8,31 @@
 #define ARTSS_BOUNDARY_MULTIGRID_H_
 
 #include "Boundary.h"
-#include "Surface.h"
+#include "BoundaryDataController.h"
 #include "Obstacle.h"
+#include "Surface.h"
 #include "../Domain.h"
 #include "../field/Field.h"
 #include "../utility/Utility.h"
-#include "BoundaryDataController.h"
 
 class Multigrid {
-public:
-    Multigrid(size_t numberOfSurfaces, Surface** surfaceList, size_t numberOfObstacles, Obstacle** obstacleList, BoundaryDataController* bdc_boundary, BoundaryDataController **bdc_obstacles);
+ public:
+    Multigrid(size_t number_of_surfaces, Surface** surface_list, size_t number_of_obstacles, Obstacle** obstacle_list, BoundaryDataController* bdc_boundary, BoundaryDataController **bdc_obstacles);
     explicit Multigrid(BoundaryDataController *bdc_boundary);
     ~Multigrid();
 
-    size_t getSize_innerList(size_t level = 0);
-    size_t getSize_boundaryList(size_t level = 0);
-    size_t getSize_obstacleList();
-    size_t *get_obstacleList();
+    size_t get_size_inner_list(size_t level = 0);
+    size_t get_size_boundary_ist(size_t level = 0);
+    size_t get_size_obstacle_list();
+    size_t *get_obstacle_list();
 
-    size_t* getInnerList_level_joined() { return m_data_MG_iList_level_joined; };
-    size_t getSize_innerList_level_joined() { return *(m_size_MG_iList_level + m_levels + 1); };
+    size_t* get_inner_list_level_joined() { return m_data_MG_inner_list_level_joined; }
+    size_t getSize_innerList_level_joined() { return *(m_size_MG_iList_level + m_levels + 1); }
     size_t getInnerList_level_joined_start(size_t level);
     size_t getInnerList_level_joined_end(size_t level);
 
-    size_t* getBoundaryList_level_joined() { return m_data_MG_bList_level_joined; };
-    size_t getSize_boundaryList_level_joined() { return *(m_size_MG_bList_level + m_levels + 1); };
+    size_t* getBoundaryList_level_joined() { return m_data_MG_bList_level_joined; }
+    size_t getSize_boundaryList_level_joined() { return *(m_size_MG_bList_level + m_levels + 1); }
     size_t getBoundaryList_level_joined_start(size_t level);
     size_t getBoundaryList_level_joined_end(size_t level);
```

Comment:
can be const

Reply:
c984e0880c80fa8fa1efe7d96bd03afc2407374c